src/Ast/AST.php

```diff
@@ -80,6 +80,21 @@ abstract class AST
     /** @var string Constant to reference `E_USER_ERROR`. */
     public const E_USER_ERROR = "\0E_USER_ERROR";
 
+    /** @var string Constant to reference `parse_url`. */
+    public const PARSE_URL = "\0parse_url";
+
+    /** @var string Constant to reference `str_replace`. */
+    public const STRING_REPLACE = "\0str_replace";
+
+    /** @var string Constant to reference `createInsecure`. */
+    public const CREATE_INSECURE = "\0createInsecure";
```

Comment:
We are typically using these constants for PHP structures. Since this is just a function name called on a class, we should leave it out of the constants here, and just hardcode it when we need it